server/controllers/comment.controller.js

```diff
@@ -1,5 +1,6 @@
-import Bluebird from 'bluebird';
+import Promise from 'bluebird';
 import mongoose from 'mongoose';
+import _ from 'lodash';
```

Comment:
lodash has a better function import so you only import what you need. For example `import map from lodash/map` which will save some space.

Reply:
👍